File: java-main/java-main/magicWhile1/p3.java
// Count digits of given number.


class Demo{

	public static void main(String[]args){

		int N=3407;
		int count=0;

		while(N!=0){
			count++;
			N=N/10;
		}
		System.out.println("Count digit of given number is " + count);
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/magic1/p3.c
/* composite number between given range*/


#include<stdio.h>
void main(){

	int num1;
	int num2;
	int count=0;

	printf("Enter the num1:");
	scanf("%d",&num1);

	printf("Enter the num2:");
	scanf("%d",&num2);

	for(int i=num1;i<=num2;i++){

		for(int j=1;j<=num2;j++){

			if(i%j==0){
				count=count+i;
			}
		}
		if(count>2){
			printf("%d",i);
		}
	}
}
